### ISSUE TITLE
Update labels: Issue Template: pre-work-template--dev

### ISSUE THREAD

**Issue by KazushiR** [hackforla/website#4478]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.
### Overview
As developers, we need to add the labels from the [Missing series](https://github.com/hackforla/website/wiki/How-to-read-and-interpret-labels#missing-series) so that we can avoid the GitHub bot taking off the labels when devs add them to issues.
### Action Items
- [x] #4682
- [x] Replace the following line
   ```
   labels: 'Feature: Onboarding/Contributing.md, prework, size: 1pt'
   ```
   with
   ```
   labels: ['Feature: Onboarding/Contributing.md', 'prework', 'size: 1pt', 'role missing', 'Complexity: Prework']
   ```
- [x] Test the updated template using these [instructions](https://github.com/hackforla/website/issues/4307#issuecomment-1549888931).  Copy the contents of that comment into a new comment in this issue, and check off the boxes as you complete the steps.  

**For PR Reviewers and Merge Team**
To review this issue, click the link below under "Link for reviewers" and verify that the labels match the updated labels string in the issue.

**Link for Reviewers**
This URL will be inserted by the dev during the test process and will be used by reviewers to preview the updated template.
URL of the issue branch on the test Repository:  [REPLACE THIS TEXT AND BRACKETS WITH THE URL]

**For PM, Merge Team, or Tech Lead**
- [ ] Once the pull request associated with this issue is approved and merged, please update and edit epic #4307 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `.github/ISSUE_TEMPLATE/pre-work-template--dev.md`
- This issue is part of epic #4307

**Comment by whitneywind** [comment]
Availability for this week: M: 6-8pm, T: 9-11am
My estimated ETA for completing this issue: Tonight or (if things don't go smoothly) tomorrow at the latest

**Comment by whitneywind** [comment]
I have a blocker. The line that the instructions say to replace seems to have been changed a bit since this issue was made, and I'm not sure if the same instructions still apply. 

The instruction say to replace:
`'labels: 'Feature: Onboarding/Contributing.md, prework, size: 1pt'`
with:
`labels: ['Feature: Onboarding/Contributing.md', 'prework', 'size: 1pt', 'role missing', 'Complexity: Missing', 'Draft']`

However, the actual current line is:
`labels: 'Complexity: Prework, Feature: Onboarding/Contributing.md, prework, size: 1pt'`

Should I still replace this line as the instructions state? Or does the way it should be updated need to be changed since the original line is different?

**Comment by whitneywind** [comment]
@KazushiR @roslynwythe Hi! I was hoping to get clarification on this blocker. It appears that either the line in my previous comment needs to be edited, or that this issue might relate to a different file, pre-work-template--design.md

**Comment by whitneywind** [comment]
Provide Update
1. Progress - posted about blocker here and in slack and tagged original authors. now waiting for clarification
2. Blockers - see blocker comments above
3. Availability -  I have a bit of time most evenings
4. ETA - within 1 day of blocker being cleared up

**Comment by t-will-gillis** [comment]
Hi @whitneywind You can proceed with the replacement line shown above 
```
labels: ['Feature: Onboarding/Contributing.md', 'prework', 'size: 1pt', 'role missing', 'Complexity: Missing', 'Draft']
```
Thank you for asking for clarification. I will remove the `Status: Help Wanted` label but for sure contact us if you still have questions.

**Comment by whitneywind** [comment]
### Instructions for testing updated issue templates:
- [x] Push your issue branch to your fork of the website repository, as described in [Section 3.1.a of CONTRIBUTING doc](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#31a-push-all-changes-to-your-issue-branch), for example:  `git push --set-upstream origin test-update-labels-1234`  
- [x] The response from the push command will include the URL for creating a pull request.  Copy/Paste the URL below for use after testing:  
(https://github.com/whitneywind/website/pull/new/update-label-template-4478)
- [x] Browse to your fork of the website repository and click Settings 
<details><summary>Screenshot of Repository Settings</summary>

![image](https://github.com/hackforla/website/assets/5314153/8ecf2e55-2187-4ab9-acf1-2fd7548291a1)

</details> 

- [x] Change the default branch to the new issue branch

<details><summary>Screenshot of Default Branch Setting</summary>

![image](https://github.com/hackforla/website/assets/5314153/945fe02e-035f-4215-9c25-1d46c209d8c0)

</details> 

- [x] Further down on the settings page under Features, check the Issues checkbox

<details><summary>Screenshot of Issues Checkbox</summary>

![image](https://github.com/hackforla/website/assets/5314153/945fe02e-035f-4215-9c25-1d46c209d8c0)

</details> 

- [x] Browse to the Issues page, then click Labels

<details><summary>Screenshot - creating labels in the test Repo</summary>

![image](https://github.com/hackforla/website/assets/5314153/208e1b07-2478-4ed3-98a1-207de58d25c5)

</details> 

- [x] Create labels corresponding to each label that was listed in the updated issue template.  The color of the labels don't have to match those on the `hackforla/website` repository, but the text of the labels must match exactly.  
- [x] From the Issues page, click `New Issue` then locate the template updated by your issue, and click `Get Started` to open the issue template. 
- [x] Copy/Paste the URL of the issue template into the Resources section of the issue, replacing the text: https://github.com/whitneywind/website/issues/new?assignees=&labels=Feature%3A+Onboarding%2FContributing.md%2CComplexity%3A+Prework%2Cprework%2Csize%3A+1pt%2Crole+missing%2CDraft&projects=&template=pre-work-template--dev.md&title=Pre-work+Checklist%3A+Developer%3A+%5Breplace+brackets+with+your+name%5D
 This will enable others to check the template.  
- [x] Verify the appearance of the issue template, including careful examination of the labels.  If there is a discrepancy between the labels listed in the issue and the labels that appear on the issue template, ask a dev lead or merge team member for help.  
- [x] Complete the PR process by visiting the URL saved from the push command
- [x] Copy these sections from the Issue into the PR 
  - [x] "For PR Reviewers and Merge Team"
  - [x] "Link for Reviewers"
  - [x] "For PM, Merge Team, or Tech Lead"
